projects/storefrontstyles/scss/cxbase/blocks/forms.scss

```diff
@@ -179,6 +179,7 @@ input[type='radio'] {
 .ng-select,
 .ng-select.ng-select-single {
   .ng-dropdown-panel {
+    box-shadow: 1px 1px 2px var(--cx-color-visual-focus);
```

Comment:
![image](https://user-images.githubusercontent.com/22354613/71254486-64cddb80-232b-11ea-8a10-9818ac7a32db.png)

Box-shadow isn't really fixing that problem imho. On the above screenshot, you can see a (input) and b (results).

1. there is a visible difference between sides and the bottom of the results
2. focus for a (input) is fine, but there is visible difference between a and b
3. sometimes a (input) looses focus and only b is focused